.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,59 @@
+name: Dependent Issues
+
+on:
+    issues:
+        types:
+            - opened
+            - edited
+            - closed
+            - reopened
+    pull_request_target:
+        types:
+            - opened
+            - edited
```

Comment:
Probably doesn't need to run for edited unless there's some issues with tickets

Reply:
fine for this but if we're ever worrying about github actions minutes then we can remove stuff like this

Reply:
this is just in case people add further "depends on #xyz" to a PR. It's counted as an edit :)